0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -605,25 +603,17 @@ The ledger plugin's ILP address prefix. This is used to determine whether a give
 
 The prefix may be configured, automatically detected, or hard-coded, depending on the ledger. For example, a Bitcoin ledger plugin may have the address hard-coded, while a [`five-bells-ledger`](https://github.com/interledger/five-bells-ledger) would use an API call to get the prefix.
 
-#### precision
```

Comment:
If the per-ledger precision is being taken out, we should specify what the expected behavior for precision is.